frontend/src/views/Game.tsx

```diff
@@ -1,10 +1,28 @@
 import Editor from '@monaco-editor/react';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { Room } from '../api/Room';
+
+type LocationState = {
+  room: Room,
+}
 
 function GamePage() {
+  const location = useLocation<LocationState>();
+  const [room, setRoom] = useState<Room | null>(null);
+
+  // Called every time location changes
+  useEffect(() => {
+    if (location && location.state && location.state.room) {
+      setRoom(location.state.room);
+    }
+  }, [location]);
+
   return (
     <div>
       Game page
+      Room:
+      {room ? room.roomId : 'No room joined'}
```

Comment:
nit: Put these text blocks in some HTML object, like a centered `h1` or something similar. (Also, do you need the "`Game page`" line?)

Reply:
I deleted `Game Page` and replaced it with the header bar in my version of this file. If we want to delete it, we could do that when we have to merge the two versions.

Reply:
I deleted it anyways since it only takes a second :)